common/src/main/java/io/netty/util/ReferenceCountUtil.java

```diff
@@ -157,6 +157,17 @@ public static void safeRelease(Object msg, int decrement) {
     }
 
     /**
+     * Returns reference count of a {@link ReferenceCounted} object. If object is not type of
+     * {@link ReferenceCounted}, {@code -1} is returned.
+     */
+    public static int refCnt(Object msg) {
+        if (msg instanceof ReferenceCounted) {
+            return ((ReferenceCounted) msg).refCnt();
+        }
+        return -1;
```

Comment:
Sure, used this style because it matches `release` and other methods in this class. Anyways - pushed update.